lib/matplotlib/table.py

```diff
@@ -403,10 +445,7 @@ def _get_grid_bbox(self, renderer):
         return bbox.inverse_transformed(self.get_transform())
 
     def contains(self, mouseevent):
-        """Test whether the mouse event occurred in the table.
-
-        Returns T/F, {}
-        """
+        # docstring inherited
```

Comment:
Note that this could additionally return e.g. which cell contained the event (in the dict).